src/main/java/org/kainos/ea/model/JobRoleGetRequest.java

```diff
@@ -0,0 +1,46 @@
+package org.kainos.ea.model;
+
+public class JobRoleGetRequest {
```

Comment:
It looks like this is the model that we are using as the response. Typically these would be called Response Object. A better name for this class might be 

```suggestion
public class JobRoleResponse {
```